### PR TITLE
Issue52.1

### PR DESCRIPTION
**Pull request by stack461**
Edited the README file to include the user instructions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,36 +1,24 @@
...
-## Learn More
-
-To learn more about Next.js, take a look at the following resources:
+## Getting Started
+The user must install [Radiance](https://www.radiance-online.org/) as well as [HDRGen](http://www.anyhere.com/) to their local machine and have access to their location. After the dependencies have been installed, install the [HDRI Calibration Interface](https://github.com/shantimorrell/HDRICalibrationTool-Capstone/actions/runs/8283470432). There will be a build for each platform. After installation, extract the contents of the zipped folder to a desired location. Open the installed folder/bundle and the first folder in here will contain the installation for your platform. Continue the installation process. This will install the application on your desktop.
 
-- [Next.js Documentation](https://nextjs.org/docs) - learn about Next.js features and API.
-- [Learn Next.js](https://nextjs.org/learn) - an interactive Next.js tutorial.
+## Use
+**Uploading Images**
```
Comment (adulbrich) [inline README.md]: Why not use `###` for a level 3 header instead of bold text?

File: README.md [hunk 2]
```diff
@@ -1,36 +1,24 @@
-This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+## Introduction
+The purpose of this application is to give the user a graphical user interface for the Radiance/HDRGen pipeline process. The program works by taking in image files and calibration files in order to return light calculated images containing radiance, luminance and color. The application is intended for interior light design researchers who are concerned with how lighting in a room affects visual discomfort.
```
Comment (adulbrich) [inline README.md]: For the introduction, you might want to check the original repository's README https://github.com/radiantlab/HDRICalibrationTool has it was vetted by Clotilde already?

File: README.md [hunk 3]
```diff
@@ -1,36 +1,24 @@
...
+**Uploading Response File and Image Information**
+Upload the response file that should have a file extension of rsp and fill in the image data for the cropping, resizing and view settings.
 
-## Deploy on Vercel
+**Uploading Calibration Files**
+Select the calibration files for the remaining fields. These should have a cal file extension.
 
-The easiest way to deploy your Next.js app is to use the [Vercel Platform](https://vercel.com/new?utm_medium=default-template&filter=next.js&utm_source=create-next-app&utm_campaign=create-next-app-readme) from the creators of Next.js.
+**Settings**
+Click on the settings tab in the left hand navigation sidebar and you should see a settings display appear. For the Radiance path, give the path to the Radiance binaries. This would be something like /usr/local/radiance/bin/ with usr being the name of the user on the computer. The HDRGen path should point to /usr/local/bin and the output should point to the location to which you want the output images to go to.
```
Comment (adulbrich) [inline README.md]: For code or file paths, you might want to use backticks `` to render it, e.g., `/usr/local/radiance/bin/`

File: README.md [hunk 4]
```diff
@@ -1,36 +1,24 @@
...
 
-You can start editing the page by modifying `app/page.tsx`. The page auto-updates as you edit the file.
-
-This project uses [`next/font`](https://nextjs.org/docs/basic-features/font-optimization) to automatically optimize and load Inter, a custom Google Font.
-
-## Learn More
-
-To learn more about Next.js, take a look at the following resources:
+## Getting Started
+The user must install [Radiance](https://www.radiance-online.org/) as well as [HDRGen](http://www.anyhere.com/) to their local machine and have access to their location. After the dependencies have been installed, install the [HDRI Calibration Interface](https://github.com/shantimorrell/HDRICalibrationTool-Capstone/actions/runs/8283470432). There will be a build for each platform. After installation, extract the contents of the zipped folder to a desired location. Open the installed folder/bundle and the first folder in here will contain the installation for your platform. Continue the installation process. This will install the application on your desktop.
```
Comment (adulbrich) [inline README.md]: In the future, the link to binaries will need to change as this points to a specific run of GitHub Actions but it should point to the latest "stable" release.

**Review by adulbrich — DISMISSED** [review]

**Comment by adulbrich** [comment]
@stack461 Once the PR is ready, I'd recommend rebasing before merging to cleanup the commit history 🥇

**Review by shantimorrell — DISMISSED** [review]
This looks like a good overview of the app!

**Review by shantimorrell — APPROVED** [review]
Looks good!

**Pull request merged**